src/Options/Applicative/Builder.purs

```diff
@@ -256,24 +250,24 @@ commandGroup g = fieldMod $ over CommandFields \p ->
 completeWith :: forall f a. HasCompleter f => Array String -> Mod f a
 completeWith = completer <<< listCompleter
 
--- | Add a bash completion action. Common actions include @file@ and
--- @directory@. See
--- <http://www.gnu.org/software/bash/manual/html_node/Programmable-Completion-Builtins.html#Programmable-Completion-Builtins>
--- for a complete list.
+-- | Add a bash completion action. Common actions include `file` and
```

Comment:
if thing you are referencing to is in same module you can do this:
```
[`file'`](#v:file') 
```
if it's in different module, you can do this:
```
[`file`](Data.Some.Module#v:file')
```

See this for example https://github.com/slamdata/purescript-pathy/blob/v7.0.0/src/Pathy/Path.purs#L136

Reply:
(Use of just ``` `file` ``` is fine, i'm fine merging this as is.)

Reply:
While I like the hyperlinking to other functions, I'd prefer to stay away from that here for two reasons. First, because I'm not familiar enough with this library to do that accurately yet. Second, because I think this problem should be addressed in `purs docs`, not here.